WORKING_GROUPS.md

```diff
@@ -434,6 +435,37 @@ Responsibilities include:
   backporting changes to these branches.
 * Define the policy for what gets backported to release streams.
 
+### [Security](https://github.com/nodejs/security-wg)
+
+The Security Working Group manages all aspects and process linked to security for Node.js.
+
+Responsibilities include:
+* Define and maintain security policies and procedures for:
+  * the core Node.js project
+  * other projects maintained by the Node.js Foundation technical group
+* Work with the Node Security Platform to bring community vulnerability data into
+  the foundation as a shared asset.
+* Set up processes and procedures and follow these to ensure the vulnerability
```

Comment:
I think most of this first sentence could be dropped and start with "Ensure that the..."